include/emp/prefab/ButtonGroup.hpp

```diff
@@ -0,0 +1,44 @@
+#ifndef EMP_BUTTON_GROUP_HPP
```

Comment:
I think we need the Empirical boilerplate pasted up here, as in all the other header files. Copy from another file and modify.
 :/